CubeScramble.py

```diff
@@ -4,20 +4,22 @@
 # Example: R, Ri or R2
 
 
-
-
-import random
-
+from random import choice
```

Comment:
You can just import choice. Cool trick huh? Either way is fine in this case. I just wanted to show you.

Reply:
Yes, good! Thanks.